packages/Responsive/Responsive.md

```diff
@@ -65,9 +67,63 @@ initialState = {
 </div>
 ```
 
-### Testing responsive behaviour
+#### Testing responsive behaviour
 
 Moving the responsive behaviour into JavaScript enables testing of the results of media queries, which is impossible with CSS-based media queries. It is straightforwards to mock or stub the result of a media query for testing environments. Here are some examples of how we incorporated unit tests for responsive behaviour in some of the TDS components:
 
 * [Button component tests](https://github.com/telusdigital/tds/blob/b2108d1074383ba887c5b87a2c3055799937fcd3/packages/Button/__tests__/Button.spec.jsx#L52-L68) and its corresponding [JSX](https://github.com/telusdigital/tds/blob/b2108d1074383ba887c5b87a2c3055799937fcd3/shared/components/BaseButton/BaseButton.jsx#L17-L34) using the matches boolean flag
 * [Tooltip component tests](https://github.com/telusdigital/tds/blob/b2108d1074383ba887c5b87a2c3055799937fcd3/packages/Tooltip/__tests__/Tooltip.spec.jsx#L56-L102) and its corresponding [JSX](https://github.com/telusdigital/tds/blob/b2108d1074383ba887c5b87a2c3055799937fcd3/packages/Tooltip/Tooltip.jsx#L81-L108) rendering a particular bubble only when the media query matches
+
+#### SASS Media Queries
+
+SASS-based media queries are available, and are recommended for static sites. This style of responsiveness provides a few benefits like:
+* Prevents sudden scaling when server-side rendered components first appear
+* Prevents the need to wrap Responsive around your component
+  * This in turn removes the requirement for separate CSS classes for different screen sizes that your application must programmatically switch between.
+
+However, using this implementation means losing the ability to programmatically change what is rendered through JSX on resize, so keep the design of your component in mind.
+
+To make use of SASS media queries through the Responsive component, first declare the following import at the top of your .scss file.
+
+```scss
+@import '~@tds/core-responsive/responsive';
+```
+
+Once this is done, you may leverage [**sass-mq**](https://github.com/sass-mq/sass-mq) style media queries to add responsive behaviour to your component.
+
+```scss
+.someClass {
+  @include mq($from: sm, $until: lg) {
+    width: 500px;
+  }
+  @include mq($from: xl) {
+    width: 900px;
+  }
+}
+
+.someOtherClass {
+  @include mq($until: md, $and: '(orientation: landscape)') {
+    width: 100%;
+    height: 100%;
+  }
+}
+```
+
+#### MQ Parameters
+
+| Parameter     |Values | Effect                                                                                                                         |
```

Comment:
This table looks great, just be sure to left-align each column using `:---` syntax: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables